src/main/java/uk/gov/hmcts/reform/slc/logging/ErrorCode.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.slc.logging;
+
+@SuppressWarnings({"squid:S1118", "HideUtilityClassConstructor"})
+public final class ErrorCode {
```

Comment:
Could we use a enum here or we could move this to logging project as this might be reused across(if this is too much for now we can leave it in this project :)

Reply:
Enum doesn't seem like best fit - would also have to call `name()`. Could move to logging but I am not sure why error code was a requirement in the first place. Back then it looked reasonable. Perhaps worth revisiting this @jasonpaige and remove if irrelevant. As of now, with current version maybe just copy over in projects. On the other hand, if it is still indeed a requirement - logging library ought to have some tweaks